GameData/Kerbalism/Parts/LifeSupport/kerbalism-lifesupportmodule.cfg

```diff
@@ -0,0 +1,189 @@
+// ============================================================================
+// kerbalism-lifesupportmodule.cfg by APlayer updated by PiezPiedPy
+// Kerbalism external Life-Support module
+//   Adds an external life support process part to Kerbalism.
+// ============================================================================
+
+
++PART[kerbalism-chemicalplant]:NEEDS[Kerbalism,ProfileDefault]:BEFORE[Kerbalism]
+{
+  @name = kerbalism-lifesupportmodule
+  @title = External ECLSS Module
+  @category = Utility
+  @description = Seeing the lack of ECLSS redundancy, mission control urgently ordered us to convert this chemical plant to an external ECLSS module. Guaranteed to explode only 147.28% as often as the original.
+  %tags = _kerbalism
+  
+  MODULE:NEEDS[FeatureReliability]
+  {
+    name = Reliability
+    type = ProcessController
+    title = ECLSS
+    redundancy = Life Support
+    repair = true // anyone
+    mtbf = 72576000 // 8y
+    extra_cost = 2.5
+    extra_mass = 0.1
+  }
+  
+  MODULE
```

Comment:
All this stuff, can this not be configured with the same config that applies it to every pod? This smell like duplication.

Reply:
@madman2003 That's a point, never thought of that, I just pulled it out of my personal changes folder as a last thought and dropped it into the branch, gave it a quick test to make sure the Humidity bit I dropped into it worked and submitted the PR.
I'll change it later today as I'm about to go to bed 😉

Reply:
This file will have to stay as it is because the pod configuration in the default profile only works on parts with a crew capacity which this part does not have.
Only other option is to move the data out of this file and copy it into the default profile but that still leaves the problem of the data appearing to be duplicated.
Unless you know of another workaround ?

Reply:
This has the added benefit of allowing people to change it in their profile.

Reply:
@madman2003 I had the same thoughts but didn't want to bloat the default profile any more than needed, I'll have a think over it, I am more towards moving it to the default profile though, so that is probably what I will end up doing.